Engine/CommandInfoCache.cs

```diff
@@ -40,9 +40,7 @@ public CommandInfo GetCommandInfo(string commandName, string aliasName = null, C
                 return null;
             }
 
-            // If alias name is given, we store the entry under that, but search with the command name
-            var key = new CommandLookupKey(aliasName ?? commandName, commandTypes);
-
+            var key = new CommandLookupKey(commandName, commandTypes);
```

Comment:
If `aliasName` is unused, maybe time to remove it from the parameters?

Reply:
Ok, done, I also removed the left-over `GetCommandInfoInternal` function that we moved into the `CommandInfoCache` a few weeks ago but forgot to delete.
I plan to do another PR with more general tidy up later btw.